bool isStarted
---
float64 left
float64 right

